composite/market-closure/src/checks/tradinghours.ts

```diff
@@ -0,0 +1,25 @@
+import { Requester } from '@chainlink/external-adapter'
+import { convertCommonKeys, ExternalCheck } from './index'
+
+const commonKeys: Record<string, string> = {
+  FTSE: 'xlon',
+  N225: 'xjpx',
+}
+
+export const thExecute: ExternalCheck = async (symbol: string) => {
+  const url = 'https://www.tradinghours.com/api/v2/status'
+
+  const market = convertCommonKeys(symbol, commonKeys)
```

Comment:
We can just write it inline and remove the `convertCommonKeys` method altogether:
```suggestion
  const market = commonKeys[symbol] || symbol
```